build

```
@@ -34,5 +44,6 @@ target/debug/merge_html
 
 # update dataset checksums
 
-git write-tree --prefix=enclone_main/test/inputs/version14/123085 > datasets_small_checksum
-git write-tree --prefix=enclone_main/test/inputs/version14 > datasets_medium_checksum
+cd enclone-data
+git write-tree --prefix=big_inputs/version14/123085 > ../datasets_small_checksum
+git write-tree --prefix=big_inputs/version14 > ../datasets_medium_checksum
```

```suggestion
git -C enclone-data write-tree --prefix=big_inputs/version14/123085 > datasets_small_checksum
git -C enclone-data write-tree --prefix=big_inputs/version14 > datasets_medium_checksum
```
`git -C` is better style than `cd`.